mobile/src/home-view.js

```diff
@@ -295,6 +303,84 @@ class HomeView extends PureComponent {
     )
   }
 
+  renderSearch = () => {
+    const newStyle = {
+      flex: 1,
+      fontSize: 18,
+      color: '#364247',
+      textAlignVertical: 'top',
+      maxHeight: 100,
+      height: Math.max(35, this.state.inputHeight),
+      paddingTop: 0
+    }
+    const androidStyle = {
```

Comment:
why different android/ios styles?  If necessary, the Platform.select can be moved up to here.